aspnet/publishing/iis.rst

```diff
@@ -1,116 +1,187 @@
 Publishing to IIS
-=============================
+=================
 
-By `Rick Anderson`_ and `Luke Latham`_
+By `Rick Anderson`_ and `Luke Latham <https://github.com/GuardRex>`_
 
-.. contents:: Sections:
-  :local:
-  :depth: 1
+- `Supported Operating Systems`_
+- `IIS Configuration`_
+- `Install the Microsoft ASP.NET Core Module`_
+- `Install .NET Core on the Server`_
+- `Deploy the Application`_
+- `Configure the Website in IIS`_
+- `Create a Data Protection Registry Hive`_
+- `Common Errors`_
+- `Additional Resources`_
 
-Install the HTTP Platform Handler
------------------------------------
+Supported Operating Systems
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Install the HTTP Platform Handler version 1.2 or higher:
+The following operating systems are supported:
 
-  - `64 bit HTTP Platform Handler <http://go.microsoft.com/fwlink/?LinkID=690721>`_
-  - `32 bit HTTP Platform Handler <http://go.microsoft.com/fwlink/?LinkId=690722>`_
+- Windows 7 and newer
+- Windows Server 2008 R2 and newer\*
 
-If you need to enable IIS, see `IIS server configuration`_.
+\*Conceptually, the IIS configuration described in this document also applies to hosting ASP.NET Core applications on Nano Server IIS, but refer to `ASP.NET Core on Nano Server <https://docs.asp.net/en/latest/tutorials/nano-server.html>`_ for specific instructions.
 
-Configure Data Protection
------------------------------------
+IIS Configuration
```

Comment:
Ah! This is one that I hope you go in one of two directions with to address one of my pet peeves about MS documentation over the last ... well ... since 1975! :smile: This doc isn't meant to be a "walkthrough" or "tutorial" as many MS docs are not meant to be walkthroughs or tutorials. **However,** that's exactly what you have with devs coming into complex new technologies (new to them notwithstanding the new-ness of .NET Core). They're looking for a fairly comprehensive end-to-end document that covers all of the basic config and that walks them through to a successfully running app.

Further, my experience with devs is that they like a few of the more notable gocthas, warnings, tips, and troublshooting points. They all love :heart:, as I do, a comprehensive and well-organized Troubleshooting section that covers all of the basic mis-steps and mis-configs. I hope you'll go in either of these two directions with this ...

1. Leave this doc with the added bits. Don't split IIS config out. Don't hack a mere handful of comman IIS goofs from the list. Let it be the "Go To" doc for publishing .NET Core on IIS. I was only half-joking when I said MS can hit $20B this year with this kind of approach: Every support contact is at least $7 (a typical industry figure that was given to my the VP of an international telecom). Keeping this doc all together as I show will very likely minimize support requests, thus save MS operating expendatures.
2. On the other hand, Yes!, indeed, get the IIS config out (at least) ... but ... also provide a proper `ASP.NET Core on IIS` tutorial. That would perhaps be even better, because we could get into `project.json` config, attributes on the `<aspNetCore>` element, have all of the Troubleshooting (including a handful of purely IIS Common Issues that devs come up against), and more "gotchas," warnings, and tips with more IIS config commentary (if desired).

Ok ... TL;DR ... I hope you'll either leave this doc as is or agree that a full tutorial is warranted. I volunteer to write it. Please don't strip this doc of IIS config and IIS-only Troubleshooting without a tutorial. MS will lose money, and I et al. will end up wasting a lot of time on JabbR helping devs with basic IIS hosting config and troublshooting. Thanks for hearing me out.